src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -112,6 +112,13 @@
     public static class ShooterConstants {
         public static final PIDFTalonSRX LEFT_MOTOR = ShooterComponents.LEFT_MOTOR;
         public static final PIDFTalonSRX RIGHT_MOTOR = ShooterComponents.RIGHT_MOTOR;
+        //TODO: Set zone limits and coefs
+        public static final double[] ZONE_LIMITS = {};
+        public static final double[] ZONE_1_COEFS = {};
+        public static final double[] ZONE_2_COEFS = {};
+        public static final double[] ZONE_3_COEFS = {};
+        public static final double[] ZONE_4_COEFS = {};
+        public static final double[] ANGLES = {};
```

Comment:
2. find a way to add this to a json. writing this by hand using excel would be hell